ci-operator/jobs/openshift/sriov-network-operator/openshift-sriov-network-operator-master_okd-scos-presubmits.yaml

```diff
@@ -0,0 +1,58 @@
+presubmits:
+  openshift/sriov-network-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master_okd-scos$
```

Comment:
do you actually have a branch `master-okd-scos` in the repo? i think this should just be `master`